hadoop-ozone/tools/src/main/java/org/apache/hadoop/ozone/shell/snapshot/SnapshotDiffHandler.java

```diff
@@ -45,6 +45,14 @@ public class SnapshotDiffHandler extends Handler {
       index = "2")
   private String toSnapshot;
 
+  @CommandLine.Option(names = {"-t", "--token"},
+      description = "token (optional) ")
+  private String token;
+
+  @CommandLine.Option(names = {"-p", "--pageSize"},
+      description = "page size (optional)")
```

Comment:
Should elaborate on what "page size" actually means.
Number of diff entries? Number of max bytes per tranfer?